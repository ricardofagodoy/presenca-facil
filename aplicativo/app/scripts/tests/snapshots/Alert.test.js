import React from 'react';
import Alert from '../../views/components/Alert';
import renderer from 'react-test-renderer';

Date.now = jest.fn(() => 1482363367071);

test('Alert show text when present', () => {

  const component = renderer.create(
    <Alert message="message" />
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test('Alert hide when text not present', () => {
    
      const component = renderer.create(
        <Alert />
      );
    
      let tree = component.toJSON();
      expect(tree).toMatchSnapshot();
    });