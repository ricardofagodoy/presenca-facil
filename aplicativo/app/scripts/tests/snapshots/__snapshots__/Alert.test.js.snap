// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert hide when text not present 1`] = `
<div
  className="center-block hidden"
>
  <div
    className="alert alert-danger center"
    role="alert"
  />
</div>
`;

exports[`Alert show text when present 1`] = `
<div
  className="center-block "
>
  <div
    className="alert alert-danger center"
    role="alert"
  >
    message
  </div>
</div>
`;
